source/fuzz/fuzzer_pass_interchange_integer_operands.h

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SOURCE_FUZZ_FUZZER_PASS_INTERCHANGE_INTEGER_OPERANDS_
```

Comment:
The name of the fuzzer pass is not that descriptive - it would be better to include SIGEDNESS somehow.

Perhaps FuzzerPassInterchangeSignednessOfIntegerOperands.

Reply:
Changed